### PR TITLE
Add contact information at request

### PR DESCRIPTION
**Pull request by Es7evam**

### CODE REVIEW

File: backend/migrations/4_request_table.up.sql [hunk 1]
```diff
@@ -8,6 +8,8 @@ CREATE TABLE IF NOT EXISTS request (
 	userid      integer NOT NULL,
 	categoryid  integer,
 	timestamp	timestamp NOT NULL,
+	telephone 	text,
```
Comment (mbotarro) [inline backend/migrations/4_request_table.up.sql]: These fields must be NOT NULL

File: backend/usecases/requestCases.go [hunk 2]
```diff
@@ -30,7 +32,24 @@ func (rc *RequestController) GetLastRequests(before time.Time, size int) ([]mode
 
 // InsertRequest inserts the given request into the databases (postgres + ES), calling the DAL package function.
 // It returns error != nil in case some error occured.
-func (rc *RequestController) InsertRequest(req models.Request) (string, error) {
+func (rc *RequestController) InsertRequest(req models.Request, telephone bool, email bool) (string, error) {
+
```
Comment (mbotarro) [inline backend/usecases/requestCases.go]: nit: unnecesarry space

Comment (Es7evam) [reply backend/usecases/requestCases.go]: Fixed at 8b17824

File: backend/migrations/4_request_table.up.sql (hunk 1 above)
Comment (Es7evam) [reply backend/migrations/4_request_table.up.sql]: Fixed at 8b17824

**Pull request merged**